step-controller/step-controller-backend/src/test/resources/step.properties

```diff
@@ -81,3 +81,11 @@ plugins.FunctionPackagePlugin.maven.repository.central.url=https://repo1.maven.o
 
 plugins.timeseries.resolution.period=1000
 plugins.timeseries.flush.period=2000
+plugins.timeseries.collections.minute.enabled=true

```

Comment:
Comment the properties out, we should test locally with default values to avoid any surprises.